### ISSUE TITLE
Add title / aria label in documents iframe

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2018]
**Source**

MC

**Describe the bug**

No title/aria label in document iframe.

**To Reproduce**

1. Go to https://godigitalmarin.org/documents/2020-cpuc-broadband-study-2525/
2. Inspect element for document iframe

**Expected behavior**

The page h1 should be the document iframe title/aria label.

**Screenshots**

<img width="1428" alt="Screen Shot 2022-04-22 at 7 54 17 AM" src="https://user-images.githubusercontent.com/1569463/164756672-fed70dc1-4510-4705-9d9b-e1fdf57a7d72.png">

**Comment by curtismchale** [comment]
@lukefretwell easy fix I just rolled it in for the next update 1.124.0 on Thursday.

**Comment by lukefretwell** [comment]
@curtismchale is this done?
